packages/entities/entities-vaults/src/components/VaultSecretPicker.vue

```diff
@@ -133,9 +133,9 @@ const { i18n: { t } } = composables.useI18n()
 
 const props = defineProps({
   config: {
-    type: Object as PropType<KonnectPluginFormConfig | KongManagerPluginFormConfig>,
+    type: Object as PropType<KonnectBaseFormConfig | KongManagerBaseFormConfig>,
     required: true,
-    validator: (config: KonnectPluginFormConfig | KongManagerPluginFormConfig): boolean => {
+    validator: (config: KonnectBaseFormConfig | KongManagerBaseFormConfig): boolean => {
```

Comment:
Using the more generic types `KonnectBaseFormConfig` and `KongManagerBaseFormConfig` instead